ts/client/features/transfer.ts

```diff
@@ -465,9 +471,39 @@ export class TransferPool implements ITransferPool {
 
 export class SimulatorPool extends OffchainTransferFactory
     implements ITransferPool {
-    private tokenID?: number;
+    private tokenID: number;
+    private initialBalance: ERC20Value;
     private feeReceiverID?: number;
-    async setTokenID() {
+
+    private readonly pubkey: PubkeyStorageEngine;
+
+    constructor(group: Group, storageManager: StorageManager) {
+        super(group, storageManager.state);
+        this.pubkey = storageManager.pubkey;
+
+        // TODO Make these configurable?
+        this.tokenID = 1;
+        this.initialBalance = USDT.fromHumanValue("100.12");
+    }
+
+    async init() {
+        for (const user of this.group.userIterator()) {
+            const state = State.new(
+                user.pubkeyID,
+                this.tokenID,
+                this.initialBalance.l2Value,
+                0
+            );
+            await this.engine.update(user.stateID, state);
+            await this.pubkey.update(user.pubkeyID, new Pubkey(user.pubkey));
+        }
+        await this.engine.commit();
+        await this.pubkey.commit();
+
+        await this.setTokenID();
```

Comment:
I moved these to `SimulatorPool` as it seemed the right place for test data setup.

Reply:
`as it seemed the right place for test data setup`... And I was wrong. Moved back to client integration test.